controllers/users-controller.js

```diff
@@ -0,0 +1,10 @@
+const { fetchAllUsers } = require("../models/user-models")
+
+function getAllUsers(request, response){
+    fetchAllUsers()
+    .then((users)=>{
+        response.status(200).send({users:users})
```

Comment:
{users} is a handy JS shorthand for when the key and value share the same name i.e.

{users} and {users:users} mean the same thing